pom.xml

```diff
@@ -73,15 +73,15 @@
             </plugin>
         </plugins>
     </build>
-    <description>This plugins adds a toolbar for common GIT actions
+    <description>This plugins adds a toolbar for common Git/GitRemote actions
 &lt;p&gt;
 &lt;img src="https://raw.github.com/markiewb/nb-git-toolbar/master/doc/screenshot1.gif"/&gt;
 &lt;/p&gt;
 
 &lt;h2&gt;Updates&lt;/h2&gt;
-&lt;h3&gt;1.1.0:&lt;/h3&gt;
+&lt;h3&gt;1.2.0:&lt;/h3&gt;

```

Comment:
Please append your changes. The changelog has to contain even the previous changes.